app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java

```diff
@@ -622,16 +629,16 @@ else if (carta.getValorTruco(s.manilha) == 14)
 		// abaixo
 		Carta tres_testedemanilha = new Carta('3', 3);
 		Carta dois_testedemanilha = new Carta('2', 3);
-		if (qcarta == DOIS
+		if (qcarta == Qualificacao.DOIS
 				&& tres_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = TRES;
-		if (qcarta == AS && tres_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = DOIS;
+			qcarta = Qualificacao.TRES;
+		if (qcarta == Qualificacao.AS && tres_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS;
 		// caso a manilha seja um 2, o A passa a ter valor de 2!
-		if (qcarta == AS && dois_testedemanilha.getValorTruco(s.manilha) == 14)
-			qcarta = DOIS;
+		if (qcarta == Qualificacao.AS && dois_testedemanilha.getValorTruco(s.manilha) == 14)
+			qcarta = Qualificacao.DOIS;
 
-		return qcarta;
+		return qcarta.value();
```

Comment:
How about delcaring this method as `private QualidadeCarta qualidadeCarta` and returning just `qcarta` here? This way, we can remove a lot of `.value()` from the callers?